code/game/objects/items/toys.dm

```diff
@@ -168,6 +168,14 @@
 		var/mob/M = loc
 		SEND_SIGNAL(M, COMSIG_CLEAR_MOOD_EVENT, "badass_antag", /datum/mood_event/badass_antag)
 	. = ..()
+	
+	
+/obj/item/toy/balloon/arrest
+	name = "arreyst balloon"
```

Comment:
Typo, "arreyst"

Reply:
It's a feature

Reply:
Arreyst was a super popular boyband.

Reply:
Changed the description to reference this.

Reply:
Well I certainly have egg on my face.